src/commands/game_commands/join.ts

```diff
@@ -0,0 +1,24 @@
+import BaseCommand, { CommandArgs } from "../base_command";
+import { GameType } from "./play";
+import { sendErrorMessage, sendInfoMessage, getUserIdentifier } from "../../helpers/discord_utils";
+import { bold } from "../../helpers/utils";
+
+export default class JoinCommand implements BaseCommand {
+    async call({ message, gameSessions }: CommandArgs) {
+        const gameSession = gameSessions[message.guildID];
+        if (!gameSession || gameSession.gameType === GameType.CLASSIC) {
+            return;
+        }
+        if (gameSession.sessionInitialized) {
+            await sendErrorMessage(message, "Game already in session", "You can only join as a participant before an elimination game has started. Please wait until the current game ends.");
+        } else if (gameSession.participants.has(message.author.id)) {
+            sendErrorMessage(message, "Player already joined", `${bold(getUserIdentifier(message.author))} is already in the game.`);
+        } else {
+            const previouslyJoinedPlayers = gameSession.scoreboard.getPlayerNames().reverse();
```

Comment:
Why reverse?

Reply:
I decided to show players in the order of most recently joined